src/pages/wishlist/wishlist.js

```diff
@@ -1,38 +1,52 @@
 import styles from "./wishlist.module.scss"
-import { productList } from "../home/sub-components/data"
+import { useWishlist } from "@providers/wishlist-provider"
+import { useBasket } from "@providers/basket-provider"
 import { ProductWrapper, ProductBody, ProductImage, ProductActions, Button } from "react-felix-ui"
 import { MdAdd } from "@icons"
 
 const Wishlist = () => {
+
+    const { WishlistState, removeFromWishlist } = useWishlist()
+    const { addToBasket } = useBasket()
+
+    const handleMoveToBasket = (item) => () => {
+        removeFromWishlist(item._id)
+        addToBasket(item)
+    }
```

Comment:
Something new for me, the syntax of the function.

Reply:
This is a currying function.